ocaml/pci/lib/pci.ml

```diff
@@ -73,8 +73,8 @@ let maybe f = function
 let scan_bus = B.pci_scan_bus
 
 let with_string ?(size=1024) f =
```

Comment:
Maybe add a comment here explaining why `string` is not suitable linking to the FAQ.
IIUC using `string` violates this rule from the FAQ: `string unsuitable for binding to C functions that write into the string,`. 

The caveats from `ocaml_string` do not apply (we do not release the runtime lock (the default) or call back into OCaml from these functions), because `string` actually does a copy, however IIUC the lifetime of the value returned by libpci is bound the the lifetime of the input parameter, which can be moved by the GC and cause problems.

Reply:
That is exactly right. I am going to steal your comment and add it as a comment